requestor/service_manager/service_wrapper.py

```diff
@@ -2,40 +2,54 @@
 
 from typing import TYPE_CHECKING
 if TYPE_CHECKING:
-    from typing import Type
-    from yapapi.executor.services import Service
+    from typing import Type, List, Any
+    from yapapi.services import Service, Cluster
 
 
 class ServiceWrapper():
-    def __init__(self, service_cls: 'Type[Service]'):
+    def __init__(self, service_cls: 'Type[Service]', start_args: 'List[Any]'):
         self.service_cls = service_cls
+        self.start_args = start_args
         self.id = self._create_id()
         self.stopped = False
-        self._service = None
+        self._cluster = None
```

Comment:
Please annotate field type (`Optional[Something]`) when initializing to `None`.

Reply:
041fc04